src/games/strategy/ui/SwingComponents.java

```diff
@@ -295,6 +297,33 @@ public static JMenu newJMenu(final String menuTitle, final KeyboardCode keyboard
   }
 
   /**
+   * Creates a new component that emulates a multiline label.
```

Comment:
Good to see this in SwingComponents. It's intended to absorb SwingUI code. It's going to grow too large one day, but for now it's more important to keep extracting the UI code from everything else.